src/components/Modals/Nfts/components/NftOverview.tsx

```diff
@@ -22,7 +22,8 @@ type NftOverviewProps = {
 export const NftOverview: React.FC<NftOverviewProps> = ({ nftItem, nftCollection }) => {
   const translate = useTranslate()
 
-  const description = nftCollection?.description || nftItem.collection.description
+  const description =
+    nftItem.description || nftItem.collection.description || nftCollection?.description
```

Comment:
Description from item, description from poor man's collection as part of the item (to be consolidated after Mercle NFT stuff when we have a normalized slice), or actual collection meta (which is fetched separately) description